recipes/otfftw/bld.bat

```diff
@@ -0,0 +1,2 @@
+if "%ARCH%"=="32" (set CPU_ARCH=i686) else (set CPU_ARCH=x86_64)
+%PKG_NAME%-%PKG_VERSION%-py%PY_VER%-%CPU_ARCH%.exe /userlevel=1 /S /FORCE /D=%PREFIX%

```

Comment:
@jakirkham the problem is building openturns with msvc; we only succeeded on building for vc10, without bindings. Also the build time would be a problem.